### DISCUSSION TITLE
Fecha de entrega Proyecto final

### DISCUSSION THREAD

**Discussion by Juansebastianrs12** [rramosp/ai4eng.v1#202] [UDEA Students]
Profe buenos dias, debido a que la universidad entro en asamblea permanente desde hace unas semanas, cual seria la nueva fecha de entrega del proyecto final, muchas gracias

**Comment by rramosp** [comment]
como comentè en la clase esta semana ... esperemos a que todo se acabe de
definir de manera definitiva por parte de la asamblea y por parte de la
facultad

On Sun, Sep 25, 2022 at 11:36 AM Juansebastianrs12 ***@***.***>
wrote:

> Profe buenos dias, debido a que la universidad entro en asamblea
> permanente desde hace unas semanas, cual seria la nueva fecha de entrega
> del proyecto final, muchas gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/202>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIN2DEH5DNQX66TV5J3WAB5SXANCNFSM6AAAAAAQVE7ZP4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
